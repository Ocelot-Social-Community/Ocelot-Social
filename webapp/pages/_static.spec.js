import { mount } from '@vue/test-utils'
import Static from './_static.vue'

const localVue = global.localVue

describe('_static', () => {
  let wrapper

  describe('mount', () => {
    const Wrapper = (page) => {
      return mount(Static, {
        mocks: {
          $t: jest.fn(),
        },
        data() {
          return {
            pageParams: {
              internalPage: {
                hasContainer: true,
                hasBaseCard: true,
                hasLoginInHeader: true,
              },
            },
          }
        },
        localVue,
      })
    }

    beforeEach(async () => {
      wrapper = await Wrapper('faq')
    })

    it('renders', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
