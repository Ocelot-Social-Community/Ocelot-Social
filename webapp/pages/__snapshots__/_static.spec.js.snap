// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_static mount renders 1`] = `
<div>
  <div>
    <div
      class="ds-space"
      style="margin-top: 16px; margin-bottom: 16px;"
    >
      <h2
        class="ds-heading ds-heading-h2"
      >
        
      
    
      </h2>
    </div>
     
    <div
      class="ds-container ds-container-x-large"
    >
      <article
        class="base-card"
      >
        <div />
         
        <!---->
      </article>
    </div>
  </div>
</div>
`;
